tests/integration.bash

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+base=${1:-http://localhost:8080/}
+base=${base%/}
+
+n=0
+skipping=false
+curl() {
+    out=$($(which curl) "$@" 2>&1);
+}
```

Comment:
Shouldn't this set the `$skipping` flag?

```suggestion
curl() {
    skipping=false
    out=$($(which curl) "$@" 2>&1);
}
```

Refs https://github.com/clue/framework-x/pull/256/files#diff-c14edd88f2d957097dcb7b8c8cf05ae8f626db8fb8127374a925795d9e64dac1R10

Reply:
Yep, I already had this change on my local machine, just forgot to push it 🙃